articles/introduction-to-neural-networks/index.md

```diff
@@ -0,0 +1,189 @@
+
+A neural network is a technology that uses network functions to understand and convert input data into the [desired output](https://deepai.org/machine-learning-glossary-and-terms/neural-network). The neural network's inspiration is by how the human brain works.
+
+The essential building block of a neural network is a perception. Perception accomplishes simple signal processing. Neural networks are pertained to solve data problems in Speech recognition, image recognition, and pattern recognition, to name a few.
+
+### How Neural Networks work
+
+Neural Networks are complex systems with artificial neurons. The neurons receive many inputs and process a [single output](https://www.upgrad.com/blog/neural-network-architecture-components-algorithms/#:~:text=Neural%20Networks%20are%20complex%20structures,input%20into%20a%20meaningful%20output).
+
+Neural networks comprise layers of neurons. The layers consist of the:
+
+- [Input layer](https://stackoverflow.com/questions/32514502/neural-networks-what-does-the-input-layer-consist-of)
+- [Multiple hidden layers](https://deepai.org/machine-learning-glossary-and-terms/hidden-layer-machine-learning#:~:text=In%20neural%20networks%2C%20a%20hidden,inputs%20entered%20into%20the%20network.)
+- [Output layer](https://www.scan2cad.com/user-manual/the-output-layer/)
+
+The input layer receives data represented by a numeric value. Hidden layers perform most computations required by the network. Finally, the output layer predicts the output.
+
+In a neural network, neurons dominate one another. A neuron is an object that holds a numeral value that is between 0 and 1. Each layer is made of neurons. Once the input layer receives data, it is redirected to the hidden layer. At the hidden layer, computation occurs, and information is sent to the neurons of the output layer. Neurons in different layers are connected through channels. Each channel is assigned a number referred to as weight.
+
+The inputs and the weights are multiplied, and their sum is sent to neurons in the hidden layer. Each neuron adds the inputs it receives to get the sum. This value then transits through the activation function.
+
+The activation function outcome decides if a neuron is activated or not. An activated neuron transfer information in the other layers through the channels. In this approach, the data gets generated on the network. In other words, this is [forward propagation](https://towardsdatascience.com/forward-propagation-in-neural-networks-simplified-math-and-code-version-bbcfef6f9250).
+
+Feed-forward propagation takes place when the hidden layer accepts the input data. Process it as per the activation function and pass it to the output. The neuron in the output layer with a higher probability projects the result.
```

Comment:
A section describing the feed-forward propagation is necessary. Also, a section on the backpropagation algorithm would be appreciated, but not necessary.